src/mlui/enums/Adabound.py

```diff
@@ -0,0 +1,225 @@
+
+
+
+import tensorflow.compat.v2 as tf
+from keras import backend
+from keras.optimizers import Optimizer
+
+class AdaBound(Optimizer):
+    """
+    Adabound optimizer implementation 
+    # References
+        - [Adaptive Gradient Methods with Dynamic Bound of Learning Rate]
+          (https://openreview.net/forum?id=Bkg3g2R9FX)
+
+
+
+    Parameters
+    -----------
+        learning_rate : float 
+            Adam learning rate (default: 1e-3)
+
+        base_lr: float >= 0.
+             Used for loading the optimizer. Do not set the argument manually.    
+
+        
+        final_lr : float
+                  final (SGD) learning rate (default: 0.1)
```

Comment:
There are unnecessary tabs at the start of the description. The description should also start with a capital letter and end with a dot.